core/egov-indexer/src/main/java/org/egov/infra/indexer/util/IndexerUtils.java

```diff
@@ -504,7 +505,9 @@ public DocumentContext addTimeStamp(Index index, DocumentContext context) {
 				log.info("NULL found in place of timestamp field.");
 				return context;
 			}
-			Date date = new Date(Long.valueOf(epochValue));
+			String ret=convertEpochToLong(epochValue);
+			Long value=Long.valueOf(ret);
+			Date date = new Date(value);
```

Comment:
perform all three operations in one line: Date date = new Date(Long.valueOf(convertEpochToLong(epochValue)))
Let's not create unecessary string values.